modules/aggregation-layers/src/utils/resource-utils.ts

```diff
@@ -20,7 +20,7 @@ export function getFloatTexture(device: Device, opts: FloatTextureOptions) {
     width = 1,
     height = 1,
     data = null,
-    unpackFlipY = true,
+    // unpackFlipY = true,
```

Comment:
Why not just delete?

Reply:
Done — At a glance thought this might have been the last reference to the prop and would keep it for 'documentation' of the parameter name for when we loop back around to fixing this function. But i see the prop is declared above on `FloatTextureOptions` too. Removed the comment!